Frontend/ui-customer/src/Flow.purs

```diff
@@ -1197,11 +1219,28 @@ homeScreenFlow = do
         homeScreenFlow
       else do
         PlaceName placeDetails <- getPlaceName lat lon gateAddress
+        let currentLocationItem = Just (dummyLocationListState{
```

Comment:
we are using dummyLocationListState to actually update the screen data? if other items are null or not expected how is it handled?

Also if required then let's rename it to just `locationListStateObj`

Reply:
`dummyLocationListState` has default/dummy value and we are only updating the values which are required here. Updated the Name to `locationListStateObj`